ZFSin/zfs/include/sys/zfs_context.h

```diff
@@ -64,7 +64,22 @@
 
 #endif
 
-#define noinline
+#ifdef ENOTACTIVE
+#undef ENOTACTIVE
+#endif
+
+#define ENOTACTIVE EOWNERDEAD
```

Comment:
I believe these should be moved to `spl/include//sys/errno.h` - I think I already have done in `rp_synth` branch, but not quite ready for commit, so just leave them be for now.